server/data/restClient.test.ts

```diff
@@ -79,6 +79,20 @@ describe('restClient', () => {
 
       expect(nock.isDone()).toBeTruthy()
     })
+    it('should not retry for 503', async () => {
+      const response = createMock<Response>({})
+      fakeApprovedPremisesApi.get('/some/path').reply(503)
+
+      const loggerSpy = jest.spyOn(logger, 'warn')
+
+      await expect(restClient.pipe({ path: '/some/path' }, response)).rejects.toThrowError(
+        'cannot GET /some/path (503)',
+      )
+
+      expect(loggerSpy).toHaveBeenCalledWith(new Error('cannot GET /some/path (503)'), 'Error calling premisesClient')
```

Comment:
Would it be worth expecting `loggerSpy` to have been called with `'Received 503, Not retrying'` ?